From 8864f611d5174e0a145ded6706a72387cfab4bec Mon Sep 17 00:00:00 2001
From: Jed <lejosnej@ainfosec.com>
Date: Thu, 9 May 2019 10:48:25 -0400
Subject: [PATCH] Revert "CP-9798: Update cgroups path"

This commit is specific to systemd.
Revert for OpenXT since we use sysvinit.

This reverts commit d92ceeba82046b5144c654f627ad846e08f1dcac.
---
 drivers/tapdisk-server.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/tapdisk-server.c b/drivers/tapdisk-server.c
index 49ecd8e..cc0d1ad 100644
--- a/drivers/tapdisk-server.c
+++ b/drivers/tapdisk-server.c
@@ -462,8 +462,8 @@ int eventfd(unsigned int initval, int flags)
 #define RESET_BACKOFF 512
 
 #define MEMORY_PRESSURE_LEVEL "critical"
-#define MEMORY_PRESSURE_PATH "/sys/fs/cgroup/memory/memory.pressure_level"
-#define EVENT_CONTROL_PATH "/sys/fs/cgroup/memory/cgroup.event_control"
+#define MEMORY_PRESSURE_PATH "/sys/fs/cgroup/memory.pressure_level"
+#define EVENT_CONTROL_PATH "/sys/fs/cgroup/cgroup.event_control"
 
 static int tapdisk_server_reset_lowmem_mode(void);
 
-- 
2.1.4

